scripts/build.d/bzip2

```diff
@@ -9,7 +9,11 @@ git clone git://sourceware.org/git/bzip2.git ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}
 pushd ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src/${pkgname} > /dev/null
 
   INSTALL_PREFIX=${DEVENVPREFIX}
+  buildcmd make.log make -f Makefile-libbz2_so
+  buildcmd make.log make clean
   buildcmd make.log make install PREFIX=${INSTALL_PREFIX} -j ${NP}
+  cp libbz2.so.1.0.8 ${DEVENVPREFIX}/lib
```

Comment:
Will the version number keep to be 1.0.8?  What will happen if the repository moves on to a new development phase?